src/test/utils.tsx

```diff
@@ -22,7 +22,7 @@ export type TestUserRole = Role | "ANONYMOUS";
 interface RenderTestEnvironmentOptions {
   role?: TestUserRole;
   modifyUser?: (u: typeof mockUser) => typeof mockUser;
-  PageComponent?: React.FC<any>;
+  PageComponent?: React.FC<any> | (React.ComponentClass<any> & WithRouterStatics<any>);
```

Comment:
Similar thing here, is this also needed?

Reply:
I did this because the Gameboard component was wrapped in `withRouter` - I've changed that now so that it's making use of `useLocation()`